src/styles/sass/partials/_header.scss

```diff
@@ -0,0 +1,68 @@
+// Header
+
+header,
+.form-section {
+    background-color: #A9B4C2 ;
+}
+h1 {
+    padding-top: 10px;;
+}
+.header-container{
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+label {
+    text-align: center;
+    justify-content: center;
+    margin-top: 300px;
+    margin-bottom: 20px;
+}
+.input-container {
+    margin: 0 auto;
+    width: 50%;
+    display: flex;
+    justify-content: center;
+}
+input {
+    border-radius: 10px;
+    border:none;
+    background-color: #EEF1EF;
+    width: 100%;
+    margin:10px auto 40px auto;
+    padding:10px 0 10px 20px;
+    flex-shrink: 2;
+}
+.fa-search {
+    z-index: 1;
+    font-size: 2rem;
+    color:#5E6572;
+}
+.search-button {
+    width: 15%;
+    height: 47px;
+    position: relative;
+    bottom: -10px;
+    left: 5px;
+    border: none;
+    border-radius: 10px;
+    background-color: #EEF1EF;
+    z-index:2;
+    &:hover {
```

Comment:
amazing that you included sass into this project, really impressive.  I was thinking on this hover you could add a focus as well for accessibility purposes.